staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go

```diff
@@ -74,6 +74,7 @@ func New() *Generator {
 			`+k8s.io/apimachinery/pkg/runtime`,
 			`k8s.io/apimachinery/pkg/apis/meta/v1`,
 			`k8s.io/apimachinery/pkg/apis/meta/v1beta1`,
+			`k8s.io/apimachinery/pkg/apis/testapigroup/v1`,
```

Comment:
this need a follow-up to remove these hardcoded package from k8s.io/code-generator

Reply:
can add a tag for protobuf .

Reply:
Yes, like for deepcopy.

Reply:
Ok, I can try send a follow-up PR to remove the hardcoded package from k8s.io/code-generator

Reply:
:+1: